frontend/src/metabase/query_builder/actions/downloading.ts

```diff
@@ -70,19 +70,31 @@ const getDatasetParams = ({
   visualizationSettings,
 }: DownloadQueryResultsOpts): DownloadQueryResultsParams => {
   const cardId = question.id();
-  const isSecureDashboardEmbedding = dashcardId != null && token != null;
+  const isQuestionInStaticEmbedDashboard = dashcardId != null && token != null;
```

Comment:
I renamed this to make it more clear, I know we don't have downloads for the full dashboard but this should be a bit clearer